### PR TITLE
NetSim: Broken image

### PR DESCRIPTION
**Pull request by islemaster**
I tried to reuse the spinner image in the dashboard.  It worked at first.  I don't know if it broke in the production environment or as a result of some dashboard change, but now I use a spinner in the app skin.

Before:
image

After:
screenshot from 2015-05-12 10 35 17

Was just caught in the last few days, hasn't been tracked in Pivotal.

### CODE REVIEW

File: apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs [hunk 1]
```diff
@@ -195,7 +195,7 @@ function writeNodeRow(nodeID, nodeName, nodeStatus, buttonType, addlClass) {
           writeHeader(i18n.outgoingConnectionRequests());
           outgoingRequestRows.forEach(function (row) {
             var outgoingStatus = i18n.lobbyStatusWaitingForOther({
-              spinner: '<img src="/assets/spinner-big.gif" />',
+              spinner: '<img src="' + assetUrl('media/turtle/loading.gif') + '" />',
```
Comment (islemaster) [inline apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs]: This asset already existed at this "turtle" location and is used by NetSim in controls.html.ejs.

Comment (breville) [inline apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs]: it seems odd to be reaching around to the turtle assets.. should this image be in a more common location?

File: apps/src/netsim/netsimGlobals.js [hunk 2]
```diff
@@ -24,10 +31,12 @@ var netsim_ = null;
 module.exports = {
 
   /**
-   * Set the root controller that can be used for global operations.
+   * Set the root controllers that can be used for global operations.
+   * @param {StudioApp} studioApp
    * @param {NetSim} netsim
    */
-  setRootController: function (netsim) {
+  setRootControllers: function (studioApp, netsim) {
```
Comment (breville) [inline apps/src/netsim/netsimGlobals.js]: does netsim now have a dependency on studioapp?

Comment (islemaster) [inline apps/src/netsim/netsimGlobals.js]: Yes, it does.  It's called out explicitly here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/netsim.js#L185

**Comment by islemaster** [comment]
Updated to point to netsim-specific loading spinner, not depend on turtle assets.

**Pull request merged**